recipes/joommfutil/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.7" %}
+
+package:
+  name: joommfutil
+  version: {{ version }}
+
+source:
+  fn: joommfutil-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/81/0e/b50101da4fc63c7dab5cc97231141c1f41871d740805fbdcf216cc83dae7/joommfutil-0.7.tar.gz
```

Comment:
Please use the canonical pypi.io link:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18